src/app/api/auth/[...nextauth]/authOptions.ts

```diff
@@ -9,152 +9,30 @@
 // SPDX-License-Identifier: EPL-2.0
 // License-Filename: LICENSE
 
-import { NextAuthOptions } from 'next-auth'
-import CredentialsProvider from 'next-auth/providers/credentials'
-import crypto from 'crypto';
-
-import { CREDENTIAL_PROVIDER } from '@/constants'
-import { HttpStatus, UserCredentialInfo } from '@/object-types'
-import AuthService from '@/services/auth.service'
-import { ApiUtils } from '@/utils'
-import { SW360User } from '../../../../../nextauth'
-import { SW360_API_URL, SW360_REST_CLIENT_ID, SW360_REST_CLIENT_SECRET } from '@/utils/env';
-
-let codeVerifier: crypto.BinaryLike;
-export const authOptions: NextAuthOptions = {
-    // Configure one or more authentication providers
-    providers: [
-        {
-            id: "sw360-backend",
-            name: "sw360 backend",
-            type: "oauth",
-            version: "2.0",
-            wellKnown: SW360_API_URL+"/authorization/.well-known/oauth-authorization-server",
-            checks: ["pkce", "state"],
-            idToken: true,
-            // Partial GH Copilot generated- start
```

Comment:
Please read Eclipse guidelines posted by @GMishx  Disclaimer should be on top after the header and one time only